### ISSUE TITLE
Search Terms doesn't work

### ISSUE THREAD

**Issue by Leinad4Mind** [ForumHulp/statistics#8]
I've installed a week ago this extension and the search terms still doesn't show anything.
I guess it is not working.

**Comment by Leinad4Mind** [comment]
I've installed cron and search results but search terms still doesn't work. :(

**Comment by ForumHulp** [comment]
Sorry for the late answer, busy with adjusting extensions for 3.1.10.
Did the cronjob of statistics run?

**Comment by Leinad4Mind** [comment]
Yes, "forumhulp.cron.statistics" runned twice. And the search results ext is working good.

**Comment by ForumHulp** [comment]
To see searchresults you need the extension. You don't need the cronstatus extension the run them.

Greetz

**Comment by Leinad4Mind** [comment]
XD you misunderstood me. My question was about search terms.
If I needed search results extension to see search terms on this statistics extension.
Don't know if it's a requirement. As the cron ext, don't know if its a requirement as the helper extension.

But no matter if I need them or not, with or without those 2 ext. Search terms doesn't work. I just wanted to figure out why. And if it required the search results extension then the problem could be of that extension. That way I would know where to search for the problem.

**Comment by Leinad4Mind** [comment]
Any hint on where to search the issue would be appreciated. :P

**Comment by ForumHulp** [comment]
You are talking about the searchterms in statistics. I was thinking about the top 10. They depend on the searchengines and the query. You can define the searchengines in ACP administer, edit searchengines

**Comment by Leinad4Mind** [comment]
On top10, on the search terms block it appears empty because the search terms doesn't show anything on statistics.

Here: http://imgur.com/a/5qqrj
